src/components/pages/Home/component/EmailSubmitForm.tsx

```diff
@@ -13,17 +15,28 @@ interface FormData {
 export default function EmailSubmitForm() {
   const { t } = useTranslation(['common', 'page-home'])
   const router = useRouter()
-  const { handleSubmit, register, formState, getFieldState } = useForm<FormData>({
+  const { handleSubmit, register, getFieldState } = useForm<FormData>({
     mode: 'onBlur',
     reValidateMode: 'onBlur',
     shouldUseNativeValidation: false,
   })
 
-  const { invalid, error, isTouched } = getFieldState('email', formState)
-
-  function submitAction(obj: FormData) {
+  const { invalid, error, isTouched } = getFieldState('email')
+  const { mutate, isPending } = useMutation({
+    mutationFn: async (email: string) => {
+      return await EmailCheckApi(email)
+    },
```

Comment:
이렇게 쓰는 것과 동일해요

```suggestion
    mutationFn: (email: string) => {
      return EmailCheckApi(email)
    },
```

Reply:
이후 작업에서는 다음과 같이 작성되었습니다.
```suggestion
    mutationFn: EmailCheckApi,
```

처음엔 type에서 문제가 있을 것 같아 저렇게 작성하였는데, 자동 추론이 된다는 것을 늦게 깨달았습니다.

Reply:
아하 넵 ㅎㅎ